language/Portuguese_Brazil/strings.po

```diff
@@ -94,206 +94,208 @@ msgstr "Desativado"
 
 #: plugins/cheat/init.lua:776
 msgid " added"
-msgstr ""
+msgstr " adicionado"
 
 #: plugins/cheatfind/init.lua:344
 msgid "Save Cheat"
-msgstr ""
+msgstr "Salvar Truque"
 
 #: plugins/cheatfind/init.lua:347
 msgid "Default"
-msgstr ""
+msgstr "Padrão"
 
 #: plugins/cheatfind/init.lua:347
 msgid "Custom"
-msgstr ""
+msgstr "Personalizado"
 
 #: plugins/cheatfind/init.lua:348
 msgid "Cheat Name"
-msgstr ""
+msgstr "Nome do Truque"
 
 #: plugins/cheatfind/init.lua:354 plugins/cheatfind/init.lua:718
 msgid "Default name is "
-msgstr ""
+msgstr "Nome padrão é "
 
 #: plugins/cheatfind/init.lua:362
 msgid "Player"
-msgstr ""
+msgstr "Jogador"
 
 #: plugins/cheatfind/init.lua:367
 msgid "Type"
-msgstr ""
+msgstr "Tipo"
 
 #: plugins/cheatfind/init.lua:373
 msgid "Save"
-msgstr ""
+msgstr "Salvo"
 
 #: plugins/cheatfind/init.lua:399
 msgid "Cheat written to "
-msgstr ""
+msgstr "Truque escrito para "
 
 #: plugins/cheatfind/init.lua:399
 msgid " and added to cheat.simple"
-msgstr ""
+msgstr " e adicionado para truque.simples"
```

Comment:
cheat.simple is a file name, and shouldn't be translated - the file name doesn't change with language